src/dev/dev.js

```diff
@@ -26,6 +19,8 @@ type State = {|
   showOverlay: boolean,
 |}
 
+const body = (document.body: any)
```

Comment:
flow makes this optional in part to support server side rendering.

Reply:
yeah, we were not handling it gracefully before either (
https://github.com/gymnastjs/gymnast/pull/404/files#diff-6cfc93b07368555038269f630aa16f54L8) but no reason not to fix it, will update